styles/index.css

```diff
@@ -1,48 +1,43 @@
 * {
     box-sizing: border-box;
+    font-family: Helvetica, sans-serif;
+}
+
+.heading-brushes,
+.heading-frames, 
+.heading-paint {
+    font-size: 32px;
+    color: white;
+    font-weight: bold;
+    min-width: 375px;
 }
 
 .banner-title {
-    background-image: url("/images/pattern.jpeg");
+    background-image: url("../images/pattern.jpeg");
```

Comment:
When I saved the code locally and made changes the background-image disappeared. I added ".." and it's back again.